src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/CellTableViewCell.cs

```diff
@@ -81,8 +81,16 @@ internal static UITableViewCell GetPlatformCell(UITableView tableView, Cell cell
 			cell.TableView = tableView;
 			var handler = cell.ToHandler(cell.FindMauiContext());
 			var renderer = (handler as CellRenderer) ?? (handler.PlatformView as CellRenderer);
+			UITableViewCell platformCell;
```

Comment:
Could simplify it in a single line:

`var platformCell = cell is not ViewCell ? renderer.GetCell(cell, reusableCell, tableView) : renderer.PlatformView;`

Reply:
@jsuarezruiz, I have made the changes as suggested